assets/wizards/popups/views/segmentation/SingleSegment.js

```diff
@@ -183,30 +205,56 @@ const SingleSegment = ( { segmentId, setSegments, wizardApiFetch } ) => {
 					</div>
 				</SegmentSettingSection>
 				<SegmentSettingSection title={ __( 'Newsletter', 'newspack' ) }>
-					<CheckboxControl
-						checked={ segmentConfig.is_subscribed }
-						onChange={ updateSegmentConfig( 'is_subscribed' ) }
-						label={ __( 'Is subscribed to newsletter', 'newspack' ) }
-					/>
-					<CheckboxControl
-						checked={ segmentConfig.is_not_subscribed }
-						onChange={ updateSegmentConfig( 'is_not_subscribed' ) }
-						label={ __( 'Is not subscribed to newsletter', 'newspack' ) }
+					<SelectControl
+						onChange={ value => {
+							value = parseInt( value );
+							if ( value === 0 ) {
+								updateSegmentConfig( {
+									is_subscribed: false,
+									is_not_subscribed: false,
+								} );
+							} else {
+								updateSegmentConfig( {
+									is_subscribed: value === 1,
+									is_not_subscribed: value === 2,
+								} );
+							}
+						} }
+						// eslint-disable-next-line no-nested-ternary
+						value={ segmentConfig.is_subscribed ? 1 : segmentConfig.is_not_subscribed ? 2 : 0 }
+						options={ [
+							{ value: 0, label: __( 'N/A', 'newspack' ) },
+							{ value: 1, label: __( 'Is subscribed', 'newspack' ) },
+							{ value: 2, label: __( 'Is not subscribed', 'newspack' ) },
+						] }
 					/>
 				</SegmentSettingSection>
 				<SegmentSettingSection
 					title={ __( 'Donation', 'newspack' ) }
 					description={ __( '(if using WooCommerce checkout)', 'newspack' ) }
 				>
-					<CheckboxControl
-						checked={ segmentConfig.is_donor }
-						onChange={ updateSegmentConfig( 'is_donor' ) }
-						label={ __( 'Has donated', 'newspack' ) }
-					/>
-					<CheckboxControl
-						checked={ segmentConfig.is_not_donor }
-						onChange={ updateSegmentConfig( 'is_not_donor' ) }
-						label={ __( "Hasn't donated", 'newspack' ) }
+					<SelectControl
+						onChange={ value => {
+							value = parseInt( value );
+							if ( value === 0 ) {
+								updateSegmentConfig( {
+									is_donor: false,
+									is_not_donor: false,
+								} );
+							} else {
+								updateSegmentConfig( {
+									is_donor: value === 1,
+									is_not_donor: value === 2,
+								} );
+							}
+						} }
+						// eslint-disable-next-line no-nested-ternary
+						value={ segmentConfig.is_donor ? 1 : segmentConfig.is_not_donor ? 2 : 0 }
+						options={ [
+							{ value: 0, label: __( 'N/A', 'newspack' ) },
```

Comment:
Same as the above comment, what about these labels?

- Both donors and non-donors
- Donors
- Non-donors

Reply:
Done In 3c5238bdd6e56efa27df563b251afad8223d2a80

Reply:
Looks like exactly what I described, except I like better what you did with the Newsletter options by omitting the "Both". Could we change the default option here to match? "Donors and non-donors"

Reply:
Makes sense, done in f466a432ffb666edf037f39a87d306389d274a9f